packages/edition-twig/patternlab-config.json

```diff
@@ -0,0 +1,144 @@
+{
+  "engines": {
+    "twig": {
+      "namespaces": [
+        {
+          "id": "atoms",
+          "paths": [
+            "source/_patterns/00-atoms"
+          ]
+        },
+        {
+          "id": "molecules",
+          "paths": [
+            "source/_patterns/01-molecules"
+          ]
+        },
+        {
+          "id": "organisms",
+          "paths": [
+            "source/_patterns/02-organisms"
+          ]
+        },
+        {
+          "id": "templates",
+          "paths": [
+            "source/_patterns/03-templates"
+          ]
+        },
+        {
+          "id": "pages",
+          "paths": [
+            "source/_patterns/04-pages"
+          ]
+        },
+        {
+          "id": "macros",
+          "paths": [
+            "source/_macros"
+          ]
+        }
+      ]
+    }
+  },
+  "cacheBust": true,
+  "cleanPublic": true,
+  "defaultPattern": "all",
+  "defaultShowPatternInfo": false,
+  "ishControlsHide": {
+    "s": false,
+    "m": false,
+    "l": false,
+    "full": false,
+    "random": false,
+    "disco": false,
+    "hay": true,
+    "mqs": false,
+    "find": false,
+    "views-all": false,
+    "views-annotations": false,
+    "views-code": false,
+    "views-new": false,
+    "tools-all": false,
+    "tools-docs": false
+  },
+  "ishViewportRange": {
+    "s": [
+      240,
+      500
+    ],
+    "m": [
+      500,
+      800
+    ],
+    "l": [
+      800,
+      2600
+    ]
+  },
+  "logLevel": "info",
+  "outputFileSuffixes": {
+    "rendered": ".rendered",
+    "rawTemplate": "",
+    "markupOnly": ".markup-only"
+  },
+  "paths": {
+    "source": {
+      "root": "./source/",
+      "patterns": "./source/_patterns/",
+      "data": "./source/_data/",
+      "meta": "./source/_meta/",
+      "annotations": "./source/_annotations/",
+      "styleguide": "dist/",
+      "patternlabFiles": {
```

Comment:
@EvanLovely woah woah woah - hold up! I’m like 99% sure we need to swap these out with out verbatim with the Twig versions of these (and add this to the monorepo here): 

I already went ahead and ported these original Twig Templates over/ updated to work with the latest CSS / JS changes in the UIKit Workshop if this helps: 
https://github.com/bolt-design-system/bolt/tree/master/packages/styleguidekit-twig-default

Reply:
I think it's working right now as this supports rendering Twig and Mustache files. I'm fine with getting Twig ones in there as well.